src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -1015,6 +1017,22 @@ public String getRecipientNames(Set<String> recipients, String courseId, String
     private void ___________methods_to_serve_local_files() {
     //========================================================================    
     }
+    
+    /**
+     * Sets the list of status messages.
+     * @param statusMessagesToUser a list of status messages that is to be displayed to the user
+     */
+    public void setStatusMessagesToUser(List<StatusMessage> statusMessagesToUser) {
+        this.statusMessagesToUser = statusMessagesToUser;
+    }
+    
+    /**
+     * Gets the list of status messages.
+     * @return a list of status messages that is to be displayed to the user
+     */
+    public List<StatusMessage> getStatusMessagesToUser() {
```

Comment:
The getter is needed so that ${data.statusMessagesToUser} can be used at various jsp files.
e.g. <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}" />

Without the getter I think the data.statusMessagesToUser will give an error.